docs/source/project_documentation/dev_guide/aac_validation.md

```diff
@@ -52,3 +59,23 @@ The internal logic of the function is up to the user, but the plugins generally
 3. If the definition under test doesn't meet the constraints of the validator, register and return an error message
 
 For more information on creating and utilizing validator plugins, please view the [Validation Plugins for Developers](validation_plugins).
+
+## Primitive Validation
+Primitive validation is AaC's way of enforcing constraints for 'primitive' types such as `integer`, `number`, `file`, `date`, `string`, and other types that don't need a separate data structure to be represented effectively. All primitive types in the core AaC language have (or will have in coming releases) corresponding primitive validations in the first party plugin [`primitive-type-check`](https://github.com/DevOps-MBSE/AaC/tree/main/python/src/aac/plugins/first_party/primitive_type_check)
+
+### Primitive Validator Plugins
+Each plugin can register multiple primitive validators, each of which provides a validation function that's used to validate a corresponding primitive type. Primitive validators are registered via the plugin function `plugin.register_primitive_validations(...)`.
+
+Each primitive validator should provide one validation function with the following signature:
+```python
```

Comment:
Missed flagging this in the earlier review, this copy paste snippet can be replaced with the literal includes also.